x-pack/plugins/lens/public/datasources/form_based/operations/definitions/terms/include_exclude_options.tsx

```diff
@@ -247,9 +256,7 @@ export const IncludeExcludeRow = ({
             options={termsOptions}
             selectedOptions={includeExcludeSelectedOptions.include}
             onChange={(options) => onChangeIncludeExcludeOptions(options, 'include')}
-            onCreateOption={(searchValue, options) =>
```

Comment:
This works great already, but I am wondering if we could make one tiny adjustment. When we have some options selected in the opposite option (for example we have Women's Clothing chosen in `include`) maybe we shouldn't display it in exclude? WDYT @MichaelMarcialis?

that could be done by modifying the options prop here in line 246 and then for 322:
```
options={termsOptions?.filter((o1) => !includeExcludeSelectedOptions.exclude?.find((o2) => o2.label === o1.label))}

<img width="420" alt="Screenshot 2024-11-05 at 10 16 09" src="https://github.com/user-attachments/assets/09b339a6-9aa8-4fb1-934d-0d81f7f78569">

Reply:
Thanks for the ping, @mbondyra! My initial instinct is to say that we should show those options in the "Exclude values" selector, even if they are already selected in the "Include values" selector (or vice versa). If the user chooses one of those values from the "Exclude values" selector that is already applied in the "Include values" selector, my expectation is that option will be automatically unselected from "Include values" and selected in "Excluded values". Does that make sense?

Doing so will prevent users having to do extra work/clicks in getting the available option they want in the opposing selector (assuming they wish to swap include/exclude).

Reply:
Thanks Michael for you input. That sounds good too and is aligned with how @VriaA implemented it right now so no changes are needed to merge this PR 🎉